mmaction/models/recognizers/recognizer3d.py

```diff
@@ -29,13 +31,28 @@ def _do_test(self, imgs):
         num_segs = imgs.shape[1]
         imgs = imgs.reshape((-1, ) + imgs.shape[2:])
 
-        x = self.extract_feat(imgs)
-        if hasattr(self, 'neck'):
-            x, _ = self.neck(x)
+        if self.test_batch is not None:
+            tot = imgs.shape[0]
+            assert num_segs == tot, ('test_batch is only compatible with '
+                                     'batch_size == 1')
+            ptr = 0
```

Comment:
`ptr` and `total` are not good names. How about `batch_point` and `total_batches` ?

Reply:
I will change it to view_ptr and total_views, I think `ptr` is widely used to represent a pointer.